SPECS/kernel-mshv/kernel-mshv.spec

```diff
@@ -11,14 +11,15 @@
 Summary:        Mariner kernel that has MSHV Host support
 Name:           kernel-mshv
 Version:        5.15.98.mshv1
```

Comment:
in the kernel-uvm SPEC we're using "kernelver" as a variable but not here. any reason why we would not want to align this between both packages?

Reply:
I'm more inclined to remove it from kernel-uvm SPEC. It just seems like unnecessary indirection, as you can already reference %{version}